client/src/components/GoBackButton/GoBackButton.module.scss

```
@@ -0,0 +1,34 @@
+@import "../../global.scss";
+
+.goBackButton {
+    width: 2.5vw;
+    height: 4vh;
```

nicht eher rem oder prozent benutzen?
```suggestion
    width: 1.5rem;
    padding-block: 1rem;
```
Werte dann noch anpassen damit sie deinen Wünschen entsprechen.